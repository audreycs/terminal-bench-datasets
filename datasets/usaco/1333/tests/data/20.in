10
686 681341882408235553
961632287055210199 674
908 102741991216141808
890 138144729944759257
419 829534498941150031
505 624489596307607256
871 609803422000753610
983533393633626409 998
212341556536790997 446
826239058008214318 864102713930342835
