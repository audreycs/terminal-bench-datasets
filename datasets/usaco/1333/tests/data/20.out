993209741119930
1426754135096851
113151972705234
155218797690766
1979795940193742
1236613061995292
700118739380921
985504402438531
476102144701390
9639
